cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -12,124 +12,249 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 //
 service PackagesService {
-  rpc GetAvailablePackages(kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesResponse) {
+  rpc GetAvailablePackageSummaries(GetAvailablePackageSummariesRequest) returns (GetAvailablePackageSummariesResponse) {
+     option (google.api.http) = {
+       get: "/core/packages/v1alpha1/availablepackagesummaries"
+     };
+   }
+  rpc GetAvailablePackageDetail(GetAvailablePackageDetailRequest) returns (GetAvailablePackageDetailResponse) {
     option (google.api.http) = {
-      get: "/core/packages/v1alpha1/packages"
+      get: "/core/packages/v1alpha1/availablepackagedetails"
     };
   }
-  rpc GetPackageRepositories(kubeappsapis.core.packages.v1alpha1.GetPackageRepositoriesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetPackageRepositoriesResponse) {
+  rpc GetPackageRepositories(GetPackageRepositoriesRequest) returns (GetPackageRepositoriesResponse) {
     option (google.api.http) = {
       get: "/core/packages/v1alpha1/packagerepositories"
     };
   }
-  rpc GetPackageMeta(kubeappsapis.core.packages.v1alpha1.GetPackageMetaRequest) returns (kubeappsapis.core.packages.v1alpha1.GetPackageMetaResponse) {
-    option (google.api.http) = {
-      get: "/core/packages/v1alpha1/packagemeta"
-    };
-  }
+
+  // TODO: remaining operations yet to be defined
+
+  // rpc GetAvailablePackageVersions(GetAvailablePackageVersionsRequest) returns (GetAvailablePackageVersionsResponse) {
+  //   option (google.api.http) = {
+  //     get: "/core/packages/v1alpha1/availablepackageversions"
+  //   };
+  // }
+  // rpc GetInstalledPackageSummaries(GetInstalledPackageSummariesRequest) returns (GetInstalledPackageSummariesResponse) {
+  //   option (google.api.http) = {
+  //     get: "/core/packages/v1alpha1/installedpackagesummaries"
+  //   };
+  // }
+  // rpc GetInstalledPackageDetail(GetInstalledPackageDetailRequest) returns (GetInstalledPackageDetailResponse) {
+  //   option (google.api.http) = {
+  //     get: "/core/packages/v1alpha1/installedpackagedetails"
+  //   };
+  // }
+  // rpc CreateInstalledPackage(CreateInstalledPackageRequest) returns (CreateInstalledPackageResponse) {
+  //   option (google.api.http) = {
+  //     post: "/core/packages/v1alpha1/installedpackages"
+  //   };
+  // }
+  // rpc UpdateInstalledPackage(UpdateInstalledPackageRequest) returns (UpdateInstalledPackageResponse) {
+  //   option (google.api.http) = {
+  //     put: "/core/packages/v1alpha1/installedpackages"
+  //   };
+  // }
+  // rpc DeleteInstalledPackage(DeleteInstalledPackageRequest) returns (DeleteInstalledPackageResponse) {
+  //   option (google.api.http) = {
+  //     delete: "/core/packages/v1alpha1/installedpackages"
+  //   };
+  // }
 }
-//
+
 // Standard request and response messages for each required function are defined
 // below for import and re-use.
 
-// GetAvailablePackages
+// -- Start definitions of the request messages -- 
```

Comment:
Wonder if it would be simpler to have the request and response together for each?

Reply:
I initially tried that order, but there is a minor downside: the annotations for building the API docs differ depending on whether they are for the request or the response (has a title and an example). So, when mixing responses and requests, one can easily omit an annotation (if using, for instance, the above definition as 'inspiration').

So, the reason for this ordering was just trying to avoid manual errors or omissions,. but, yep, semantically it makes more sense to have them together.

See what you think, no strong opinion on my side :P

Reply:
Yeah, same - no strong opinion. If you've found it safer in this order to ensure the correct annotations, let's do that.